### PR TITLE
Added get_ULEZ function

### PR DESCRIPTION
**Pull request by wengraf**
Added a get_ULEZ function.  This is useful for Air Quality work of course, but it also shows DVLA-format model name, which neither the DVSA MOT API nor the DVLA VES API do.

### CODE REVIEW

File: R/ulez.R [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#' Download DVLA-based vehicle data from the TfL API using VRM.
+#'
+#' @section Details:
+#' This function takes a a character vector of vehicle registrations (VRMs) and returns DVLA-based vehicle data from TfL's API, included ULEZ eligibility.
+#' It returns a data frame of those VRMs which were successfully used with the TfL API.  ULEZ-exempt vehicles will not have vehicle details returned - they will simply be marked "exempt".
+#'
+#' Be aware that the API has usage limits.  The function will therefore limit API calls to 50 per minute.
```
Comment (Robinlovelace) [inline R/ulez.R:7]: Suggest adding a line before this saying something along the lines of:

```r
#' An API key is needed to access this data.
#' See [here](url) for details on getting the key.
#' You can add it to your environment with the following command:
#' 
#' `usethis::edit_r_environ()`
#' 
#' Then add a new lines as follows to the file:
#' ULEZ=paste-your-api-key-here
#' ...
```

**Comment by Robinlovelace** [comment]
Great work @wengraf and well spotted. First comment: I suggest adding a link on where to get the API key - see suggestion in-line here: https://github.com/ropensci/stats19/pull/198/files

**Review by Robinlovelace — COMMENTED** [review]
Definitely worthwhile. Unsure why tests are failing but big :+1: from me on a place to live for this.

File: R/ulez.R [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#'
+#' @param vrm A list of VRMs as character strings.
+#'
+#' @export
+#' @examples
+#' \donttest{
```
Comment (Robinlovelace) [inline R/ulez.R:13]: I suggest adding this line next:

```r
#' apikey = Sys.getenv("ULEZ")
```

Comment (Robinlovelace) [reply R/ulez.R:13]: That should make the tests pass.

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Minor suggested changes.

File: R/ulez.R (hunk 1 above)
Comment (wengraf) [reply R/ulez.R]: Good point, @Robinlovelace, but you can access 50 per minute without an API key, so the code is based around that - getting the most data you can get without the friction of registering.

File: R/ulez.R (hunk 2 above)
Comment (wengraf) [reply R/ulez.R:13]: Actually, it was a typo that still had "apikey" in the example when the function requires no api key.

File: R/ulez.R (hunk 1 above)
Comment (Robinlovelace) [reply R/ulez.R]: OK but no harm in linking to the site. 50/min is pretty good!

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
Happy with this. Ready to go from your perspective @wengraf ? Will plan to merge and write a news item tomorrow. Heads-up also @agila5, @layik and @publicHealthDataGeek in case of interest. Comments welcome.

**Comment by wengraf** [comment]
Yes, all looks OK with me.  I'd seen some people asking about VRM-lookup for safety work, and between MOT API and the TfL API's no-key tier, I think you get a lot.  I was thinking about adding in some code to make the ULEZ function work faster if you had an API key (and 50/min without), but I'm prioritising adding in another function to access the DVLA VES API too.  Between those three, you'll find about as much as you possibly could about a number plate without (i) paying or (ii) getting disclosive.

**Pull request merged**